app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+    for human in people:
+        person = Person(human["name"], human["age"])
+        if "wife" in human and human["wife"] is not None:
```

Comment:
You can write just "human['wife']" instead of "human["wife"] is not None" because bool(None) is False.